### DISCUSSION TITLE
Is there a program to calculate bbox(xyxy xyxy) to (x,y,w,h,r)

### DISCUSSION THREAD

**Discussion by oszn** [open-mmlab/mmrotate#652] [General]
Is there a program to calculate bbox(xyxy xyxy) to (x,y,w,h,r)

**Comment by yangxue0827** [comment]
> Is there a program to calculate bbox(xyxy xyxy) to (x,y,w,h,r)

https://github.com/open-mmlab/mmrotate/blob/main/mmrotate/core/bbox/transforms.py#L221
